core/java/iesi-rest-without-microservices/src/test/java/io/metadew/iesi/server/rest/script/ScriptControllerTest.java

```diff
@@ -67,220 +74,385 @@ void getAll2ResultsAndSerializationTest() throws Exception {
         ScriptDto scriptDto0 = ScriptDtoBuilder.simpleScriptDto("1stScript", 0);
         ScriptDto scriptDto1 = ScriptDtoBuilder.simpleScriptDto("1stScript", 1);
         List<ScriptDto> scriptDtoList = Stream.of(scriptDto0, scriptDto1).collect(Collectors.toList());
-        given(scriptDtoService.getAll(new ArrayList<>(), false)).willReturn(scriptDtoList);
+        Pageable pageable = PageRequest.of(0, 20);
+        Page<ScriptDto> page = new PageImpl<>(scriptDtoList, pageable, 2);
+        given(scriptDtoService.getAll(pageable, null, false)).willReturn(page);
```

Comment:
null as a List parameter feels wrong

Reply:
Without the default value, the controller calls the service that way.
Then, the service takes care to create the empty array. (And null check are done in the repository too)

I can bring back the defaultValue in the controller if you find it better.

Reply:
I brought default value back and thus change for new ArrayList()